modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/H2Utils.java

```diff
@@ -131,17 +132,82 @@ public class H2Utils {
     /**
      * The default precision for a char/varchar value.
      */
-    static final int STRING_DEFAULT_PRECISION = Integer.MAX_VALUE;
+    public static final int STRING_DEFAULT_PRECISION = Integer.MAX_VALUE;
+
+    /**
+     * The default precision for a binary/varbinary value.
+     */
+    public static final int BINARY_DEFAULT_PRECISION = Integer.MAX_VALUE;
 
     /**
      * The default precision for a decimal value.
      */
-    static final int DECIMAL_DEFAULT_PRECISION = 65535;
+    public static final int DECIMAL_DEFAULT_PRECISION = 65535;
 
     /**
      * The default scale for a decimal value.
      */
-    static final int DECIMAL_DEFAULT_SCALE = 32767;
+    public static final int DECIMAL_DEFAULT_SCALE = 32767;
+
+    /**
+     * The default precision for a boolean value (see {@link ValueBoolean}).
+     */
+    public static final int BOOLEAN_DEFAULT_PRECISION = ValueBoolean.PRECISION;
+
+    /**
+     * The default precision for a byte value (see {@link ValueByte}).
+     */
+    public static final int BYTE_DEFAULT_PRECISION = 3;
+
+    /**
+     * The default precision for a short value (see {@link ValueShort}).
+     */
+    public static final int SHORT_DEFAULT_PRECISION = 5;
+
+    /**
+     * The default precision for a integer value (see {@link ValueInt}).
+     */
+    public static final int INTEGER_DEFAULT_PRECISION = ValueInt.PRECISION;
```

Comment:
I see the difference between Ignite and H2 type system precision, check org.gridgain.internal.h2.value.TypeInfo
can we align them ? Or just use H2 one ?

Reply:
Sorry, I don't quite understand your suggestion.

In TypeInfo I see the same precision for int:
```
TYPE_INT = new TypeInfo(Value.INT, ValueInt.PRECISION, 0, ValueInt.DISPLAY_SIZE, null);
```

Reply:
> Or just use H2 one ?

Are you suggesting to use  constants from H2 classes?

They are mostly package private. Are you suggesting to make them public and don't copy values to H2Utils?

Reply:
oh i see ... 
ValueLong#PRECISION is public but 
ValueDecimal#DEFAULT_PRECISION is package private ...